### PR TITLE
Compute Generation PR 

### PR DESCRIPTION
**Pull request by haagha**
## Description

https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/756 (Partial implementation)

Set-AzVMRunCommand
Get-AzVMRunCommand
Remove-AzVMRunCommand
Set-AzVmssVMRunCommand   
Remove-AzVmssVMRunCommand
Get-AzVmssVMRunCommand

## Checklist

- [x ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [x ] For public API changes to cmdlets:
    - [x ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute.Autorest/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+### AutoRest Configuration
+> see https://aka.ms/autorest
+``` yaml
+branch: 7b19bbd8ee63fa724edf5c780b63ae038312d2b1
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/runCommands.json
```
[the remark this answers is not in the document]
Comment (isra-fel) [reply src/Compute/Compute.Autorest/README.md:38]: After this change please run autorest and build-module.ps1 again to make sure everything is OK

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Compute/Compute.Autorest/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  # Remove set cmdlet
+  # Remove following verbs
+  #- select: command
+  #  where:
+  #    verb: Export|Convert|Install|Set
+  #  remove: true
+  # Remove existing cmdlets
+  #- select: command
+  #  where:
+  #    subject: VirtualMachine|AvailabilitySet|DedicatedHost|Image|Operation|ProximityPlacementGroup|RestorePoint|RestorePointCollection|Usage|#CapacityReservationGroup|CapacityReservation
```
[the remark this answers is not in the document]
Comment (haagha) [reply src/Compute/Compute.Autorest/README.md]: fixed

File: src/Compute/Compute.Autorest/examples/Set-AzVmssVMRunCommand.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+### Example 1: {{ Add title here }}
```
[the remark this answers is not in the document]
Comment (haagha) [reply src/Compute/Compute.Autorest/examples/Set-AzVmssVMRunCommand.md]: fixed

**Review by isra-fel — APPROVED** [review]
Looks good

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hold on for #16307

**Review by isra-fel — APPROVED** [review]
As discussed with @haagha we will not wait for #16311

**Pull request merged**